drs_client/client.py

```diff
@@ -0,0 +1,95 @@
+from drs_client.models import AccessURL, DrsObject, PostDrsObject
+from pydantic import BaseModel
+import requests
+import json
+
+
+class DRSClient():
+    """Client to communicate with a GA4GH DRS instance.
+    Supports additional endpoints defined in DRS-filer (add URL).
+
+    Args:
+        host: Base URL of DRS instance.
+        port: Port at which DRS instance can be accessed.
+        base-path: Path at which DRS endpoints can be accessed.
+        token: Bearer token for authentication
+    
+    Attributes:
+        url: URL to DRS endpoints, composed of `host`, `port` and `base_path`,
+            e.g.,"https://my-drs.app:8080/ga4gh/drs/v1"
+        token: Bearer token for authentication
+    """
+
+    def __init__(
+        self, 
+        host= 'http://0.0.0.0', 
+        port = '8080', 
+        base_path= 'ga4gh/drs/v1',
+        token = None
+    ) -> None:
+        self.url = f"{host}:{port}/{base_path}"
+        self.token = token
+
+    def get_object(self, object_id):
```

Comment:
Please add typing information for all return values (for this method and all following).